extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.css

```diff
@@ -16,6 +16,15 @@ amp-story-shopping-attachment a {
   text-decoration: none !important;
 }
 
+amp-story-shopping-attachment button { 
```

Comment:
Should we make this `i-amphtml-amp-story-shopping-pdp-details-header {` to limit the scope of this selector?